app/controllers/redirections_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RedirectionsController < ApplicationController
+  def next
+    redirection = Redirection.find_by(slug: params[:slug])
```

Comment:
So is this what you're picturing:

Current webring:

```
gabebw.com         added: 10.31.2015 ⤓
alexfox.me         added: 11.02.2015 ⤴
adarsh.io          added: 11.03.2015 ⤴
```

(`⤴` represents next link direction, `⤓` to the bottom)

If I link to the webring, it would look for my slug or referring URL (however we architect it), and create it if it didn't exist. Then it would tack me on to the end, pointing toward the last known site in the ring. So everything would then look like:

```
gabebw.com         added: 10.31.2015 ⤓
alexfox.me         added: 11.02.2015 ⤴
adarsh.io          added: 11.03.2015 ⤴
edwardloveall.com  added: 11.07.2015 ⤴
```

Is that right?